src/logistration/RegistrationPage.jsx

```diff
@@ -247,8 +256,8 @@ class RegistrationPage extends React.Component {
             <StatusAlert
```

Comment:
Can we simple remove this status alert because on successful registration eventually we are going to redirect learner to the dashboard or the next URL if available?

This was just added to show some success during hackathon.

Reply:
Sure

Reply:
Done